src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php

```diff
@@ -65,19 +66,28 @@ protected function doSend(MessageInterface $message): SentMessage
         $post = [
             '$type' => 'app.bsky.feed.post',
             'text' => $message->getSubject(),
-            'createdAt' => (new \DateTimeImmutable())->format('Y-m-d\\TH:i:s.u\\Z'),
+            'createdAt' => \DateTimeImmutable::createFromFormat('U', time())->format('Y-m-d\\TH:i:s.u\\Z'),
         ];
         if ([] !== $facets = $this->parseFacets($post['text'])) {
             $post['facets'] = $facets;
         }
 
+        $options = $message->getOptions()?->toArray() ?? [];
```

Comment:
With this line, the PR will also allow the user to add any kind of options and they are sent to Bluesky. Is that intentional?

Reply:
I've reproduced what is made for other bridges like Mastodon:

https://github.com/symfony/symfony/blob/eccdbea19f78a1c140f0f838ae7165717b855e14/src/Symfony/Component/Notifier/Bridge/Mastodon/MastodonTransport.php#L72

Reply:
I understand that this was copied. 

Do we want the user to be able to provide any type of setting? Some of course will be overwritten. Like "repo" and "collection" etc.

Reply:
Is it a problem ? I the user add an unknown option it will be not processed by the transport no ?

Reply:
Do you suggestion to remove the `toArray` method an use a `getMedia` instead ?

Reply:
I just wanted to make sure it was intentional. 

> the user add an unknown option it will be not processed by the transport no ?

Im not sure. The server may throw a 400. One can argue if we should be flexible (and allow users to shoot themselves in the foot) or strict. 

> Do you suggestion to remove the toArray method an use a getMedia instead ?

An alternative would be 
```
$userOptions = $message->getOptions()?->toArray() ?? [];
```
And then build our `$options` are without arbitrary values from the user. 

-----------

Im happy to merge this PR as is.